server/util.go

```diff
@@ -209,9 +209,7 @@ func (t *slowLogTxn) Commit() (*clientv3.TxnResponse, error) {
 
 func sliceClone(strs []string) []string {
 	data := make([]string, 0, len(strs))
-	for _, str := range strs {
-		data = append(data, str)
-	}
+	data = append(data, strs...)
```

Comment:
maybe `copy` is more suitable here.

Reply:
Yes, you are right.

Reply:
This function maybe can be remove.

Reply:
Yes. It's unused. Please help to remove it.